### ISSUE TITLE
Big Debuff stopped working most recent update 10.10

### ISSUE THREAD

**Issue by Chicken-Stormscale** [jordonwow/bigdebuffs#347]
After updating my raid frames would no longer show any HoTs at all. 
De-activated Big Debuff and my HoTs showed again. 
Rolled back to 10.8 version and HoTs would show but only max 3 even though increase to 6 was checked. 
Other dispellable debuffs also were not showing with Grievious mechanic active.

**Comment by peoii** [comment]
Just ran a PF M+ with the same issue:.  Bugsack reported as:

```
44684x FrameXML\CompactUnitFrame.lua:1523: Usage: SpellIsPriorityAura(spellID)
[string "=[C]"]: in function `SpellIsPriorityAura'
[string "@FrameXML\CompactUnitFrame.lua"]:1523: in function <FrameXML\CompactUnitFrame.lua:1521>
[string "=(tail call)"]: ?
[string "@BigDebuffs\BigDebuffs-v10.10.lua"]:1414: in function <BigDebuffs\BigDebuffs.lua:1413>
[string "@BigDebuffs\BigDebuffs-v10.10.lua"]:1481: in function `func'
[string "@FrameXML\AuraUtil.lua"]:43: in function <FrameXML\AuraUtil.lua:38>
[string "@FrameXML\AuraUtil.lua"]:58: in function `ForEachAura'
[string "@BigDebuffs\BigDebuffs-v10.10.lua"]:1469: in function <BigDebuffs\BigDebuffs.lua:1448>
[string "=[C]"]: in function `CompactUnitFrame_UpdateAuras'
[string "@FrameXML\CompactUnitFrame.lua"]:96: in function <FrameXML\CompactUnitFrame.lua:45>

Locals:
(*temporary) = "Autorecreation"
```

**Comment by jordonwow** [comment]
fixed #348
